workflows/metatdenovo.nf

```diff
@@ -236,6 +237,26 @@ workflow METATDENOVO {
 
     FEATURECOUNTS_CDS ( ch_featurecounts)
     ch_versions       = ch_versions.mix(FEATURECOUNTS_CDS.out.versions)
+    //
+    // MODULE: Collect featurecounts output counts in one table
+    //
+    
+    COLLECT_FEATURECOUNTS ( FEATURECOUNTS_CDS.out.counts.map { it[1] })
+    ch_cds_counts = COLLECT_FEATURECOUNTS.out.counts
+    ch_versions = ch_versions.mix(COLLECT_FEATURECOUNTS.out.versions)
+    
```

Comment:
This row contains trailing white spaces, according to github actions (EditorConfig).